sys-fs/squashfs_tools/squashfs_tools-4.6.1.recipe

```diff
@@ -73,8 +77,24 @@ BUILD()
 	make $MAKE_ARGS $jobArgs
 }
 
+
 INSTALL()
 {
 	cd squashfs-tools
 	make install $MAKE_ARGS INSTALL_DIR=$commandBinDir INSTALL_MANPAGES_DIR=$manDir/man1
+
+	# Install new mime-type
+	mkdir -p $dataDir/mime_db/application
+	rc $portDir/additional-files/x-squashfs.rdef -o x-squashfs.rsrc
+	resattr -O -o $dataDir/mime_db/application/x-squashfs x-squashfs.rsrc
+	rm x-squashfs.rsrc
+
+	# Create expander rules
+	mkdir -p $dataDir/expander/rules
+	echo "\"application/x-squashfs\"	.sqfs	\"unsquashfs -d '' -lls %s\"	\"unsquashfs -d . %s\"" \
+		> $dataDir/expander/rules/squashfs
+	echo "\"application/x-squashfs\"	.sfs	\"unsquashfs -d '' -lls %s\"	\"unsquashfs -d . %s\"" \
+		>> $dataDir/expander/rules/squashfs
+	echo "\"application/x-squashfs\"	.squashfs	\"unsquashfs -d '' -lls %s\"	\"unsquashfs -d . %s\"" \
+		>> $dataDir/expander/rules/squashfs
```

Comment:
This doesn't seem to contain anything that depends on a recipe variable. Why not add another file to additional-files and just copy it?